Lab_1.m

```diff
@@ -0,0 +1,94 @@
+%1 
+
+d = random('Normal', 0, 1, [1,100000]);
+histogram(d,50,'DisplayStyle',"stairs",'LineWidth',3)
+xlim([-5,5])
+
+histogram(d,50,'DisplayStyle',"stairs",'LineWidth',3,"Normalization","pdf")
+hold on 
+x = linspace(5.95,6.05,1000);
+pd = makedist("Normal","mu",6.,"sigma",0.01);
```

Comment:
Also, "mu"(mean value) and "sigma" (sigma value) should be changed to mu=0 and sigma = 1 (i.e. makedist("Normal","mu",0,"sigma",1))